Tests failed on OS with Silicon processor

1. Get MacBook with Silicon processor
1. Install requirements: `uv sync --all-groups --all-extras`
1. Run tests: `nox -e test`

Result:

```python
_________________________________________________________________________________ ERROR collecting examples/postgres_example.py _________________________________________________________________________________
ImportError while importing test module '/Users/t0rr/PycharmProjects/PyrateLimiter/examples/postgres_example.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
../../.local/share/uv/python/cpython-3.10.19-macos-aarch64-none/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
examples/postgres_example.py:13: in <module>
    from psycopg_pool import ConnectionPool as PgConnectionPool
.venv/lib/python3.10/site-packages/psycopg_pool/__init__.py:7: in <module>
    from .pool import ConnectionPool
.venv/lib/python3.10/site-packages/psycopg_pool/pool.py:22: in <module>
    from psycopg import errors as e
.venv/lib/python3.10/site-packages/psycopg/__init__.py:9: in <module>
    from . import pq  # noqa: F401 import early to stabilize side effects
.venv/lib/python3.10/site-packages/psycopg/pq/__init__.py:116: in <module>
    import_from_libpq()
.venv/lib/python3.10/site-packages/psycopg/pq/__init__.py:108: in import_from_libpq
    raise ImportError(
E   ImportError: no pq wrapper available.
E   Attempts made:
E   - couldn't import psycopg 'c' implementation: No module named 'psycopg_c'
E   - couldn't import psycopg 'binary' implementation: No module named 'psycopg_binary'
E   - couldn't import psycopg 'python' implementation: dlopen(/usr/local/lib/libpq.dylib, 0x0006): tried: '/usr/local/lib/libpq.dylib' (mach-o file, but is an incompatible architecture (have 'x86_64', need 'arm64e' or 'arm64e.v1' or 'arm64' or 'arm64')), '/System/Volumes/Preboot/Cryptexes/OS/usr/local/lib/libpq.dylib' (no such file), '/usr/local/lib/libpq.dylib' (mach-o file, but is an incompatible architecture (have 'x86_64', need 'arm64e' or 'arm64e.v1' or 'arm64' or 'arm64')), '/usr/local/Cellar/postgresql/13.3/lib/libpq.5.13.dylib' (mach-o file, but is an incompatible architecture (have 'x86_64', need 'arm64e' or 'arm64e.v1' or 'arm64' or 'arm64')), '/System/Volumes/Preboot/Cryptexes/OS/usr/local/Cellar/postgresql/13.3/lib/libpq.5.13.dylib' (no such file), '/usr/local/Cellar/postgresql/13.3/lib/libpq.5.13.dylib' (mach-o file, but is an incompatible architecture (have 'x86_64', need 'arm64e' or 'arm64e.v1' or 'arm64' or 'arm64'))
```

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -71,7 +71,7 @@ dev = [
     "httpx",
     "aiohttp",
     "requests",
-    "psycopg[binary]>=3.2.9; sys_platform == 'win32' or sys_platform == 'darwin'"
+    "psycopg[binary]>=3.2.10; sys_platform == 'win32' or sys_platform == 'darwin'"
 ]
 
 

